brainstormforce/ultimate-addons-for-gutenberg
# Improvement : Units-wise min/max option for Range and Responsive Slider.

### Description

- Added option to add min/max by unit
- eg. For PX I want max 1600 min 100 and same for % I want max 100 min 10 now it is possible.
- We just need to pass the limit min/max attribute to the Range and Responsive slider.
- eq. `limitMin={ { 'px': 100, '%': 10 } }
					limitMax={ { 'px': 1500, '%': 100 } }`
### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- For Responsive Slider - Drag and drop TOC
- Go to the General > Content Tab > Check Gap Between Lists setting for PX max is 1600px and for % max is 100%.
- For Range - Drag and drop Info-box
- Go to Style Tab > Icon/Image tab > Size setting for PX max is 1600px and for EM max is 100em

### Checklist:
